.eslintrc

```diff
@@ -0,0 +1,60 @@
+{
+  "extends": [
+    "eslint:recommended",
+    "plugin:react/recommended",
+    "prettier",
+    "universe"
```

Comment:
What do you think about this..?
```suggestion
   "universe/node",
    "universe/web"
```

Reply:
https://github.com/AMP-SCZ/dpdash/pull/719/commits/f41bdf37ba018948b4fe335f58d8270451592b42